types/rdfjs__express-handler/index.d.ts

```diff
@@ -23,9 +24,13 @@ interface BaseIriFromRequest {
     (req: Request): Promise<string> | string;
 }
 
+interface Formats {
+    parsers: SinkMap<EventEmitter, Stream>;
+    serializers: SinkMap<Stream, EventEmitter>;
+}
```

Comment:
I keep this local here because I want to avoid the dependency on formats-common. The new types are for v3 which is ESM but this package is still commonjs

Duplicating the interface here makes it independent of formats package